examples/demo-react-native/app.js

```diff
@@ -10,19 +10,43 @@ import {
   StyleSheet,
   Text,
   View,
-  TouchableOpacity
+  TouchableOpacity,
+  DatePickerIOS
 } from 'react-native';
 
 class example extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      greeting: undefined
+      goToScreen: undefined,
+      greeting: undefined,
+      chosenDate: new Date()
     };
+    this._setDate = this._setDate.bind(this);
   }
+
+  _setDate(newDate) {
+    this.setState({
+      chosenDate :newDate
+    })
+  }
+
+  
+
   render() {
     if (this.state.greeting) return this.renderAfterButton();
-    return (
+    return this.state.goToScreen === 'datePicker' ? (
+      <View style={{flex: 1, paddingTop: 20, justifyContent: 'center', alignItems: 'center'}}>
+        <DatePickerIOS testID = {'CustomDatePicker'}
+          style = {styles.datePicker}
+          date={this.state.chosenDate}
+          onDateChange={this._setDate}
+        />
+        <TouchableOpacity testID='back_button' onPress={() => {this.setState({goToScreen: false})}}  >
```

Comment:
I don't think we need a back button. We always call `reloadReactNative` after every test.

Reply:
This is not the place, move it to `detox/test` project